dll/win32/shell32/folders/CRegFolder.cpp

```diff
@@ -22,110 +22,104 @@
 
 WINE_DEFAULT_DEBUG_CHANNEL (shell);
 
-HRESULT CALLBACK RegFolderContextMenuCallback(IShellFolder *psf,
-                                              HWND         hwnd,
-                                              IDataObject  *pdtobj,
-                                              UINT         uMsg,
-                                              WPARAM       wParam,
-                                              LPARAM       lParam)
-{
-    if (uMsg != DFM_INVOKECOMMAND || wParam != DFM_CMD_PROPERTIES)
-        return SHELL32_DefaultContextMenuCallBack(psf, pdtobj, uMsg);
+#define DEFAULTSORTORDERINDEX 0x80 // The default for registry items according to Geoff Chappell
```

Comment:
I would prefer it if this was backed by a link to the article, (maybe with a web.archive link)

Reply:
I'll just add a link to the top of the file since the entire concept is based on his work (RIP).